pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -115,7 +115,7 @@ let
   ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
     buildPackages.stdenv.cc
     pythonForBuild
-  ] ++ optionals (stdenv.cc.isClang && (enableLTO || enableOptimizations)) [
+  ] ++ optionals (stdenv.cc.isClang && (!stdenv.hostPlatform.useAndroidPrebuilt or false) && (enableLTO || enableOptimizations)) [
```

Comment:
`stdenv.cc.isClang` is very suspicious. Why would CPython linking LLVM have anything to do with what C compiler is used?

This is a problem for another day, though. Not this PR's fault.